src/app/pages/test-report-dashboard/components/review-feedback/review-feedback.ts

```diff
@@ -31,6 +31,7 @@ export class ReviewFeedback {
     this.form.get('feedback').setValidators([Validators.required, Validators.maxLength(950)]);
 
     this.formControl.patchValue(this.feedback);
+    this.formControl.markAsDirty();
```

Comment:
We should be doing in similar fashion to the top level components in the app. TestReportDashboard should run code like this.
```
Object.keys(this.form.controls).forEach((controlName: string) => this.form.controls[controlName].markAsDirty());
```
AC 1 says
```
If the review has no value, then a red cross and red border are added to that section on the test dashboard
```
Examples of how to do this should be able to be seen on most if not all pages of app, use WRTC for example